patroni/ha.py

```diff
@@ -1156,7 +1162,7 @@ def post_recover(self):
     def cancel_initialization(self):
         logger.info('removing initialize key after failed attempt to bootstrap the cluster')
         self.dcs.cancel_initialization()
-        self.state_handler.stop('immediate')
+        self.state_handler.stop('immediate', stop_timeout=self.master_stop_timeout())
```

Comment:
Here we just want to make sure that postgres isn't running. IMO it would still make sense to pass non-zero value here. It would theoretically help with the case when postgres doesn't respond to immediate shutdown.

Reply:
Make sense. Should we give precedence to the stop timeout set in config and override it only if its null?